From 60bdfcd7416746bf5393f1d54589ba699f9b07b4 Mon Sep 17 00:00:00 2001
From: Pankaj Bharadiya <pankaj.laxminarayan.bharadiya@intel.com>
Date: Fri, 23 Feb 2018 14:53:01 +0530
Subject: [PATCH 1465/2367] ASoC: Intel: Skylake: Remove unused link_cpr_params
 variable

In cnl_sdw_bra_pipe_cfg_pb(), link_cpr_params variable serves no useful
purpose hence remove its code reference.

Change-Id: Ica980b766003b591886f00827fcaf18f95cabc4a
Acked-by: Vaibhav Agarwal <vaibhav.agarwal@intel.com>
Signed-off-by: Pankaj Bharadiya <pankaj.laxminarayan.bharadiya@intel.com>
---
 sound/soc/intel/skylake/skl-messages.c | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/sound/soc/intel/skylake/skl-messages.c b/sound/soc/intel/skylake/skl-messages.c
index 04926b5..9b5c2b8 100644
--- a/sound/soc/intel/skylake/skl-messages.c
+++ b/sound/soc/intel/skylake/skl-messages.c
@@ -374,7 +374,7 @@ static int cnl_sdw_bra_pipe_cfg_pb(struct skl_sst *ctx,
 {
 	struct bra_conf *bra_data = &ctx->bra_pipe_data[mstr_num];
 	struct skl_pipe *host_cpr_pipe = NULL;
-	struct skl_pipe_params host_cpr_params, link_cpr_params;
+	struct skl_pipe_params host_cpr_params;
 	struct skl_module_cfg *host_cpr_cfg = NULL, *link_cpr_cfg = NULL;
 	struct skl_module *host_cpr_mod = NULL, *link_cpr_mod = NULL;
 	int ret;
@@ -532,8 +532,6 @@ static int cnl_sdw_bra_pipe_cfg_pb(struct skl_sst *ctx,
 
 	memcpy(link_cpr_cfg, host_cpr_cfg,
 			sizeof(struct skl_module_cfg));
-	memcpy(&link_cpr_params, &host_cpr_params,
-			sizeof(struct skl_pipe_params));
 
 	link_cpr_cfg->id.instance_id = 2;
 	link_cpr_cfg->id.pvt_id = skl_get_pvt_id(ctx,
-- 
2.7.4

